packages/trader/src/Modules/Contract/Components/AccumulatorsStats/__tests__/accumulators-stats.spec.tsx

```diff
@@ -13,8 +15,10 @@ const mock_connect_props = {
 jest.mock('Stores/connect.js', () => ({
     __esModule: true,
     default: 'mockedDefaultExport',
-    connect: () => Component => props => Component({ ...props, ...mock_connect_props }),
+    connect: () => (Component: React.FC<TAccumulatorsStatsProps>) => (props: TAccumulatorsStatsProps) =>
```

Comment:
```suggestion
     connect:
        () =>
        <T,>(Component: T) =>
            Component,
```
@henry-deriv according to binary-link.spec.tsx, this is how we have to type the connect in tests